### PR TITLE
Here we go again

### PR DESCRIPTION
**Pull request by andrewbolster**

### CODE REVIEW

**Review by paddycarey — APPROVED** [review]
:shipit:

**Comment by SideSwipe9th** [comment]
Looks good to me, but I don't have write access to merge the PR!

**Review by SideSwipe9th — APPROVED** [review]
Looks good to me.

File: index.html [hunk 1]
```diff
@@ -53,72 +55,130 @@
...
+                    return "It has been " + daysSinceGov + " days since Northern Ireland had a government. There have been " + currentRecordDays - daysSinceGov + " days since passing Belgium";
+                }
+            };
+        })();
+
+        var salaryCount = (function () {
+            var dailyBaseSalaryFrom2020 = 51000 / 365; // http://www.niassembly.gov.uk/your-mlas/members-salaries-and-expenses/salaries-and-expenditure-rates-2016-2021/
+
+            var salaryPerSecond = (((dailyBaseSalaryFrom2020 / 24) / 60) / 60);
+            var noOfMLAs = 90;
```
Comment (frankloan-whs) [inline index.html:144]:
```suggestion
            var noOfMLAs = 0;
```

Comment (SideSwipe9th) [reply index.html:144]: AFIK the MLA count isn't reducing despite the resignation, so the number is still needed for the salary calculations for costs incurred while NI is without a government.

Comment (cconliffe-r7) [reply index.html:144]: Is the salary actually going to be paid out to anyone?

**Pull request merged**